task09-reactjs/src/App.js

```diff
@@ -0,0 +1,20 @@
+import './App.css';
+import MainNavigation from './components/MainNavigation';
+import AllCards from './components/AllCards';
+import Searcher from './components/Searcher';
+import Footer from './components/Footer';
+
+function App() {
```

Comment:
Let's use arrow functions for this: `const App = ()=>{}`

Reply:
Done